cvat-ui/src/components/create-task-page/create-task-content.tsx

```diff
@@ -242,7 +242,6 @@ export default class CreateTaskContent extends React.PureComponent<Props, State>
                     <Button
                         loading={loading}
                         disabled={loading}
-                        type='danger'
```

Comment:
What about setting it as a 'primary'?

And we have the same situation in upload a model form. So, to be consistent we should change button type there too.